wherehows-backend/conf/application.conf

```diff
@@ -49,6 +49,14 @@ db.wherehows.url = ${WHZ_DB_URL}
 db.wherehows.username = ${WHZ_DB_USERNAME}
 db.wherehows.password = ${WHZ_DB_PASSWORD}
 
+# Hibernate HIKARICP connection driver name
+# HikariCP is an open source JDBC connection pooling library
+hikaricp.dataSourceClassName=${WHZ_DB_DSCLASSNAME}
```

Comment:
Maybe provide a hard-coded default and make it overridable using ${?WHZ_DB_DSCLASSNAME}. like db.wherehows.drvier